GliaWidgets/Sources/Component/UnreadMessageIndicator/UnreadMessageIndicatorStyle.Accessibility.swift

```diff
@@ -4,13 +4,16 @@ extension UnreadMessageIndicatorStyle {
         /// Accessibility label.
         public var label: String
 
+        /// - Parameters:
+        ///   - label: Accessibility label.
```

Comment:
I would remove "." from here because it's just a very short phrase, not a sentence.
```suggestion
        ///   - label: Accessibility label
```

Reply:
fixed [2f6c133](https://github.com/salemove/ios-sdk-widgets/pull/887/commits/2f6c133f6a3b48578ec46a81c10e20675231e135)